dec_2023/Team_Leader_Harsha_Harod/Gala_Token_Analysis.py

```diff
@@ -0,0 +1,671 @@
+# Gala_Token_analysis.py
+import streamlit as st
+from PIL import Image
+import pandas as pd
+
+
+
+def overview_page():
+
+    
+    img=Image.open("gala_logo.jpg")
```

Comment:
You have your image file `gala_logo.jpg` in `Assets/` directory

And we run the application from where `Gala_Token_Analysis.py`, so the path given should be `Assets/gala_logo.jpg` instead.

```suggestion
    img=Image.open("Assets/gala_logo.jpg")
```

Reply:
Got it ,will correct it,actually when I runned the code using "view in codespace",I again hade to change the path to "workspace/...../assets/...."

Reply:
I have updated sir,please do check.Thankyou!